.buildkite/core.rayci.yml

```diff
@@ -26,7 +26,7 @@ steps:
     commands:
       - bazel run //ci/ray_ci:test_in_docker -- //python/ray/tests/... //python/ray/dag/... python/ray/autoscaler/v2/... core 
         --workers "$${BUILDKITE_PARALLEL_JOB_COUNT}" --worker-id "$${BUILDKITE_PARALLEL_JOB}" --parallelism-per-worker 3
-        --except-tags debug_tests,asan_tests,post_wheel_build,xcommit
+        --except-tags debug_tests,asan_tests,post_wheel_build,xcommit,manual
```

Comment:
why add these `manual`? `manual` is a bazel built-in tag, and won't be included in `...` or `:all` queries.

Reply:
Hmm I basically added this because serverless tests had this tag in their config & when I didn't add this, it ran those tests with manual tags in bazel build for some reasons (and after I added this, it didn't run).

Reply:
which one? and in which test job?

@can-anyscale , could you verify and fix the test selection so that it always exclude manual tests in the rayci wrapper?

Reply:
>  why test_dashboard fails here? Do you think this has never been running? https://buildkite.com/ray-project/premerge/builds/9888#018b6b75-00be-44d4-bcee-5b954771a670.

ah, it is the `test_dashboard` one I think

Reply:
yeah test_dashboard, and there was one other test I removed from this PR (because the test file was deleted). 

Lmk if you want me to delete this manual tag. I can follow up!